stream-chat-android-offline/build.gradle

```diff
@@ -45,10 +45,8 @@ android {
     }
 }
 
-kapt {
-    arguments {
-        arg("room.incremental", "true")
-    }
+ksp {
+    arg("room.useNullAwareTypeAnalysis", "false")
```

Comment:
Room [2.4.0-beta02](https://developer.android.com/jetpack/androidx/releases/room#2.4.0-beta02) introduced a new `TypeConverter analyzer` that takes nullability information in types into account. We should turn this option off for avoiding the type error.

Reply:
I'll add a task to remove this as flag will be removed in the future